packages/@apphosting/adapter-nextjs/src/utils.ts

```diff
@@ -96,23 +98,28 @@ export async function generateOutputDirectory(
   await move(standaloneDirectory, outputBundleOptions.outputDirectory, { overwrite: true });
 
   const staticDirectory = join(nextBuildDirectory, "static");
-  const publicDirectory = join(appDir, "public");
   await Promise.all([
     move(staticDirectory, outputBundleOptions.outputStaticDirectory, { overwrite: true }),
-    movePublicDirectory(publicDirectory, outputBundleOptions.outputPublicDirectory),
+    moveResources(appDir, outputBundleOptions.outputBundleAppDir),
     generateBundleYaml(outputBundleOptions, nextBuildDirectory, rootDir),
   ]);
   return;
 }
 
-// move public directory to apphosting output public directory
-async function movePublicDirectory(
-  publicDirectory: string,
-  appHostingPublicDirectory: string,
-): Promise<void> {
-  const publicDirectoryExists = await exists(publicDirectory);
-  if (!publicDirectoryExists) return;
-  await move(publicDirectory, appHostingPublicDirectory, { overwrite: true });
+// Move all files and directories directory to apphosting output directory.
```

Comment:
Just to make sure I understand - we are simply copying over everything from the application directory now? These files will stay on the server right (since they're not being specified as static or public assets)?

Reply:
Yeah all the application dir files are being copied over now to be used on the server.